src/pages/BrowseSchemes.svelte

```diff
@@ -25,9 +26,15 @@
   import { gjScheme, map } from "../stores";
   import type { Scheme as GjScheme } from "../types";
 
-  onMount(() => {
+  // TODO Remove after the input data is fixed to plumb correct authority names.
+  let authorityNames: Set<string> | null = null;
+
+  onMount(async () => {
     // For govuk components. Must happen here.
     initAll();
+
+    let geojson = await getAuthoritiesGeoJson();
```

Comment:
This is an extra file for us to load on this page, slows down startup slightly. So, fixing the upstream problem in the combo data should be important.